roles/backup/vars/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 deployment_type: "eda"
-_postgres_image: quay.io/sclorg/postgresql-13-c9s
+_postgres_image: quay.io/sclorg/postgresql-15-c8s
```

Comment:
is there a reason to switch to the c8s image rather than sticking on the c9s one ?